src/applications/keyboard/preload.ts

```diff
@@ -0,0 +1,22 @@
+import { IpcRenderer } from 'electron'
+
+declare global {
+  namespace NodeJS {
+    interface Global {
+      ipcRenderer: IpcRenderer
+    }
+  }
+
+  interface Window {
+    global: Window
+  }
+}
+
+const ipcRenderer = require('electron').ipcRenderer
+
+process.once('loaded', () => {
+  window.global = window.global || (window as any)
+  global.ipcRenderer = ipcRenderer
+})
+
+export {}
```

Comment:
Why ? What does it do ?

Reply:
Hmmmmm right... I think it was to be able to use a `declare` statement above so it's considered as a module,  but since there already is an import statement I might be able to remove it